code/game/objects/items/twohanded.dm

```diff
@@ -613,6 +613,28 @@
 	if(wielded)
 		. = ..()
 
+// ENERGY CHAINSAW!!
+/obj/item/twohanded/required/chainsaw/energy
+	name = "energy chainsaw"
+	desc = "As awesome as both an energy sword and chainsaw combined."
+	icon_state = "energy_chainsaw_off"
+	force_on = 32
+	armour_penetration = 100
+	var/knockdown = 1
```

Comment:
unused var, you can just get rid of this

Reply:
I thought 'var/knockdown = 1' lets the chainsaw actually knock people down?